Gemfile.lock

```diff
@@ -865,4 +865,4 @@ RUBY VERSION
    ruby 2.7.5p203
 
 BUNDLED WITH
```

Comment:
Could you review this?

Reply:
yep! this is fixed in this [PR](https://github.com/decidim/decidim/pull/8749/commits) .
After it is merged, I'll update this PR and this issue will be fixed.